index.html

```diff
@@ -14,11 +14,13 @@
     </style>
   </head>
 
-<script src="/socket.io/socket.io.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.0/socket.io.js"></script>
```

Comment:
I didn't have socket.io.js locally, so I made this change to pull it from a remote site like you did for jquery below.